### ISSUE TITLE
Create Language Filters

### ISSUE THREAD

**Issue by ghosh** [iamsankalp/signages.co#18]
We should really consider adding a language filter soon. Up to 4 languages already. It is worth also considering showing only the english signages by default and hiding the others behind a filter, just to cut down the noise.

@iamsankalp @jaydeepgajera  @mailtokamalnayan  @nimkarkedar Thoughts?

**Comment by iamsankalp** [comment]
More like tabs in a way? I like the idea about showing only English by default because: 
1) It'll save on bandwidth.
2) The pattern gets communicated and we'll mostly have available signages in all languages anyway.

**Comment by nimkarkedar** [comment]
No, different language discoverability needs to be upfront. A guy opening
in Haryana should not see english at all.

Kedar http://nimkarkedar.com
Everything is abracadabra. Everything.

**Comment by ghosh** [comment]
I disagree. In that sense, someone opening in Bangalore would only see Kannada. 

We cannot make the assumption that anyone opening from a particular region is looking for signages in the local language of that region **only**.

We should always show English and maybe along with that the local language of the region.

**Comment by nimkarkedar** [comment]
Yes, i agree, but all I am saying is, Signages are available in multiple
languages should be upfront.
May be we extent the header to show tabs of 5 to 6 languages.

Kedar http://nimkarkedar.com
Everything is abracadabra. Everything.

**Comment by mailtokamalnayan** [comment]
I'm working on that. Will keep you guys posted. 

Meanwhile, can we move this to a CMS? It's a huge pain in the ass to implement small change in the grid.

Kamal

**Comment by nimkarkedar** [comment]
Kedar http://nimkarkedar.com
Everything is abracadabra. Everything.

**Comment by nimkarkedar** [comment]
Fuck, I missed Gujarati. Sorry MaaGajera.

Can we also have a email group to chat? May be slack. For one parent
company called "????" Don't get me started.

Kedar http://nimkarkedar.com
Everything is abracadabra. Everything.
